docs/book/src/introduction.md

```diff
@@ -27,8 +27,9 @@ Also, this [guide](https://github.com/kubernetes/registry.k8s.io/tree/main/docs/
 
 <h1>ClusterAPI documentation versions</h1>
 
-This book documents ClusterAPI v1.6. For other Cluster API versions please see the corresponding documentation:
+This book documents ClusterAPI v1.7. For other Cluster API versions please see the corresponding documentation:
```

Comment:
```suggestion
This book documents ClusterAPI v1.8. For other Cluster API versions please see the corresponding documentation:
```

This is not true anymore because the release-1.7 branch is cut?!

Note: we can't yet add the link for release-1.7 because the url is not yet up.